### PR TITLE
[TS] LPS-167473

### PR DESCRIPTION
**Pull request by jcampoy**
https://issues.liferay.com/browse/LPS-167473

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jcampoy** [comment]
ci:test:relevant

**Comment by jcampoy** [comment]
ci:test:sf

File: portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java [hunk 1]
```diff
@@ -97,7 +98,8 @@ private static boolean _isEmpty(HttpServletRequest httpServletRequest) {
 	private static final Class<?>[] _CLASSES_HIDE_DEFAULT_ERROR_MESSAGE =
 		ArrayUtil.append(
 			new Class<?>[] {
-				NoSuchLayoutException.class, UserPasswordException.class
+				NoSuchLayoutException.class, NoSuchGroupException.class,
```
Comment (lfbesada) [inline portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java:101]: could you please sort alphabetically the classes?

**Review by lfbesada — CHANGES_REQUESTED** [review]
Hi @jcampoy 
Thanks for the fix!
I would like to ask only for a minor sf change, could you please sort alphabetically the classes?
Thanks in advance!
Regards

File: portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java (hunk 1 above)
Comment (jcampoy) [reply portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java]: Hi Lourdes!! of course! I have updated de branch. Thank you and have a good Friday :)

**Review by lfbesada — APPROVED** [review]
Thanks for the fix!

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:reevaluate:1287401_5029

**Comment by lfbesada** [comment]
ci:forward

**Comment by ozysouza** [comment]
Just started reviewing :)

**Comment by ozysouza** [comment]
The failures are unrelated to this PR. Please see https://github.com/brianchandotcom/liferay-portal/pull/125785

**Pull request closed without merging**